.travis.yml

```diff
@@ -16,26 +16,23 @@ env:
 matrix:
   include:
     - name: 'PHPStan'
-      php: 7.2
+      php: 7.4
       env: RUN_PHPSTAN="TRUE"
   fast_finish: true
 
 cache:
     directories:
         - $HOME/.composer/cache
 
-install:
-  - if [ $RUN_PHPSTAN == "TRUE"  ]; then composer require --dev phpstan/phpstan:^0.12; fi
-
 before_script:
   - composer update --prefer-source $PREFER_LOWEST
   - PHP_BIN=$(phpenv which php)
   - sudo $PHP_BIN -S localhost:80 -t $TRAVIS_BUILD_DIR/tests/www 2>/dev/null &
 
 script:
-  - if [ $RUN_PHPSTAN == "FALSE"  ]; then ./bin/php-cs-fixer fix --dry-run --diff; fi
-  - if [ $RUN_PHPSTAN == "FALSE"  ]; then ./bin/phpunit --configuration tests/phpunit.xml $WITH_COVERAGE; fi
-  - if [ $RUN_PHPSTAN == "TRUE"  ]; then php ./bin/phpstan analyse -c phpstan.neon lib; fi
+  - if [ $RUN_PHPSTAN == "FALSE" ]; then php vendor/bin/php-cs-fixer fix --dry-run --diff; fi
+  - if [ $RUN_PHPSTAN == "FALSE" ]; then php vendor/bin/phpunit --configuration tests/phpunit.xml $WITH_COVERAGE; fi
+  - if [ $RUN_PHPSTAN == "TRUE" ]; then php vendor/bin/phpstan analyse -c phpstan.neon lib tests; fi
```

Comment:
what do you think about putting this paths into the https://github.com/sabre-io/http/blob/master/phpstan.neon instead?

that way the travis.yml will not be different for this line across reposes

Reply:
yes, we could do that
let's merge the current stuff for now, and then I can make another round of PRs to `skel` then each repo so we get rid of the `lib tests`